users/vieting/experiments/switchboard/hybrid/feat/experiments.py

```diff
@@ -213,10 +213,14 @@ def run_specaug_scf():
 
     nn_args = get_nn_args_baseline(
         nn_base_args={
-            "scf": dict(
-                returnn_args=dict(batch_size=7000, specaug_mask_sorting=True),
+            "scf_mask_size-4": dict(
+                returnn_args=dict(batch_size=3500, specaug_mask_sorting=True, mask_divisor=4, extra_args=dict(accum_grad_multiple_step = 4)),
```

Comment:
What is the reason for switching to batch size 3500 with accumulation 4? I would expect that the change of the SpecAugment mask size does not influence the memory consumption.